rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -121,11 +122,11 @@ components:
       required: true
       schema:
         $ref: '#/components/schemas/Version'
-    Id:
-      name: Id
+    DatasetId:
+      name: datasetId
       in: path
       description: >
-        The ID of the resource to index using the attached file.
+        The ID of the dataset to index using the attached metacard.
```

Comment:
❓ Is this supposed to say `using the attached metacard` at the end?

Reply:
That phrase is used multiple times throughout this API. I agree it could be improved, but I think that's outside the scope of this PR.